packages/menus/src/containers/MenuContainer.js

```diff
@@ -150,18 +150,22 @@ class MenuContainer extends ControlledComponent {
     };
   }
 
-  componentDidMount() {
+  attachClickOutsideHandler() {
     this.getDocuments().forEach(doc => {
-      doc.addEventListener('mousedown', this.handleOutsideMouseDown);
+      doc.addEventListener('mousedown', this.handleOutsideMouseDown, true);
```

Comment:
It's always friendly to provide a quick inline comment reminder for boolean parameters: i.e. `doc.addEventListener(..., true /* useCapture*/);